docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md

```diff
@@ -41,35 +47,94 @@ Regardless of the alert level, you create and configure an alert as follows:
 
   Templates are available for alerts related to Yugabyte Platform operations, YugabyteDB operations, as well as YSQL and YCQL performance.<br>
 
-  The **Active** field allows you to define the alert as initially active or inactive.<br>
+  Most of the template fields are self-explanatory. The following fields are of note:
 
-  The **Threshold** field allows you to define the value (for example, number of milliseconds, resets, errors, nodes) that must be reached in order to trigger the alert.<br>
+  - The **Active** field allows you to define the alert as initially active or inactive.<br>
 
-  The **Destination** field allows you to select one of the previously defined recipients of the alert. 
+  - The **Threshold** field allows you to define the value (for example, number of milliseconds, resets, errors, nodes) that must be reached by the metric in order to trigger the alert.<br>
+
+  - The **Destination** field allows you to select one of the previously defined recipients of the alert. For more information, see [Define alert destinations](#define-alert-destinations).
 
 - Click **Save**.
 
-- Click **Alert Destinations** to define a new destination, view details of an existing destination, edit or delete a destination, as per the following illustration: <br><br>
+## Define notification channels
+
+In Yugabyte Platform, a notification channel defines how an alert is issued (via an email, a Slack message, a webhook message, or a PagerDuty message) and who should receive it.<br>You can create a new channel, as well as modify or delete an existing one as follows: 
+
+- Navigate to **Alert Configurations > Notification Channels**, as per the following illustration: 
+
+  <br><br>
+
+  ![Notification channel](/images/yp/config-alerts7.png)
+
+  <br><br>
+
+- To create a new channel, click **Add Channel** and then complete the **Create new alert channel** dialog shown in the following illustration:<br>
+
+  ![New channel](/images/yp/config-alerts6.png)
+
+  <br><br>
+
+  If you select **Email** as a notification delivery method, perform the following: 
```

Comment:
"do the following" is shorter, and more non-native speaker friendly...

Reply:
As a non-native speaker, I disagree with the second argument: to some of us, "do" sounds like a command